src/components/CreateToken/Form.tsx

```diff
@@ -143,23 +181,20 @@ const CreateTokenForm: React.FC<CreateTokenFormProps> = ({ tokenType }) => {
               readOnly={tokenType === TokenType.NFT}
               placeholder="100,000"
               min={1}
-              max={200000000}
```

Comment:
I wonder what are the implications of someone creating too many tokens. Will the chain get jammed with txs, since 1 token = 1 tx right?

Reply:
@piggydoughnut yup.

Reply:
Actually it depends. Reading this again 1 token isn’t 1 tx, since one tx can create multiple tokens fungible between themselves.